Umbraco-Cloud/Getting-Started/Baselines/Upgrading-Child-Projects/index.md

```diff
@@ -13,4 +13,29 @@ From this page you will have an overview af all the Child Projects this Baseline
 4. Follow the process as the upgrades are pushed to the child projects one by one
 5. When the Child Projects *turn green* the upgrade was a success
 
-![Upgrade Child Projects](images/manage-baseline-children.gif)
\ No newline at end of file
+![Upgrade Child Projects](images/manage-baseline-children.gif)
+
+## Deploying Minor upgrades to Child projects
+
+To do a minor upgrade of a Baseline project and its Child projects, the initial task is to run the minor upgrade on the baseline it self. This is done as you'd expect in the Umbraco Cloud portal by clicking the "Upgrade available!" button on the Development environment. 
+
+Following [Minor upgrades](../../Upgrades/#minor-version-upgrades) guide and process will leave the Baseline project upgraded to the latest version, while all the children are untouched for now.
+
+Once the upgrade has been verified on the Baseline project, and you've made sure everything works as expected, you should go to the **Manage Child Projects** page on the Baseline. 
+
+On this page you'll notice that all the children will be marked with pending upgrades. At this point you are able to select one or more projects to upgrade. 
```

Comment:
```suggestion
On this page you'll notice that all the children now have an available upgrade. At this point you are able to select the projects to upgrade.

At first any pending changes made on the Baseline will be deployed to the child site. Once the changes have been deployed, the child site will be upgraded to the same version as the Baseline site. All the products (CMS, Courier to Deploy and Forms) will be upgraded.
```

I believe it's important to note down that any pending changes made on the Baseline but not yet on the Child site will be at first deployed/synced and the upgrade will start afterwards.